### PR TITLE
docs(weekly): add this week in databend 119

### PR DESCRIPTION
**Pull request by PsiACE**

### CODE REVIEW

**Comment by PsiACE** [comment]
cc @soyeric128 for review

File: blog/2023-11-12-databend-weekly-119.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+For more information, please check out the resources below.
+
+- [Docs | External Function](https://databend.rs/doc/sql-commands/ddl/external-function/)
+- [databend-udf · PyPI](https://pypi.org/project/databend-udf/)
+
+## Highlights
+
+We have also made these improvements to Databend that we hope you will find helpful:
+
+- Added `translate` function.
```
Comment (soyeric128) [inline blog/2023-11-12-databend-weekly-119.md:82]:
```suggestion
- Introduced `translate` function.
```

File: blog/2023-11-12-databend-weekly-119.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+### Support `CURRENT_TIMESTAMP` Function
+
+The `CURRENT_TIMESTAMP` function returns the current date and time, with an optional precision parameter to control the number of digits after the decimal point in the output.
+
+Reference example as follows:
```
Comment (soyeric128) [inline blog/2023-11-12-databend-weekly-119.md:95]:
```suggestion

```

File: blog/2023-11-12-databend-weekly-119.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+- Added `translate` function.
+- Added support for `task_history` system table.
+- `MERGE INTO` now supports using table directly as data source. 
+- `SHOW DROP TABLE` now supports filtering and `LIMIT` options.
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+### Support `CURRENT_TIMESTAMP` Function
```
Comment (soyeric128) [inline blog/2023-11-12-databend-weekly-119.md:91]:
```suggestion
### New Function: CURRENT_TIMESTAMP
```

File: blog/2023-11-12-databend-weekly-119.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+- [Docs | External Function](https://databend.rs/doc/sql-commands/ddl/external-function/)
+- [databend-udf · PyPI](https://pypi.org/project/databend-udf/)
+
+## Highlights
+
+We have also made these improvements to Databend that we hope you will find helpful:
+
+- Added `translate` function.
+- Added support for `task_history` system table.
```
[the remark this answers is not in the document]
Comment (PsiACE) [reply blog/2023-11-12-databend-weekly-119.md:83]: got it.

File: blog/2023-11-12-databend-weekly-119.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+- [Docs | External Function](https://databend.rs/doc/sql-commands/ddl/external-function/)
+- [databend-udf · PyPI](https://pypi.org/project/databend-udf/)
+
+## Highlights
+
+We have also made these improvements to Databend that we hope you will find helpful:
+
+- Introduced `translate` function.
+- Added support for `task_history` system table.
+- `MERGE INTO` now supports using table directly as data source.
```
Comment (soyeric128) [inline blog/2023-11-12-databend-weekly-119.md:84]:
```suggestion
- `MERGE INTO` now supports using table directly as a data source. 
```

File: blog/2023-11-12-databend-weekly-119.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+- [Docs | External Function](https://databend.rs/doc/sql-commands/ddl/external-function/)
+- [databend-udf · PyPI](https://pypi.org/project/databend-udf/)
+
+## Highlights
+
+We have also made these improvements to Databend that we hope you will find helpful:
+
+- Introduced `translate` function.
+- Added support for `task_history` system table.
```
Comment (soyeric128) [inline blog/2023-11-12-databend-weekly-119.md:83]:
```suggestion
- Introduced a new system table named `task_history` .
```

**Pull request merged**